ui/src/app/services/swagger-fetch-clients.ts

```diff
@@ -157,6 +157,8 @@ export function registerApiClient<T extends BaseAPI>(
 registerApiClient(
   ConfigApi,
   new (class extends ConfigApi {
+    basePath: string;
```

Comment:
Anonymous class needed properties declared or the constructor would show an error.